api/src/utils/redact-object.ts

```diff
@@ -111,7 +111,18 @@ export function getReplacer(replacement: Replacement, values?: Values) {
 		? Object.entries(values).filter(([_k, v]) => typeof v === 'string' && v.length > 0)
 		: [];
 
+	const seen = new WeakSet();
+
 	return (_key: string, value: unknown) => {
+		// Skip circular values
```

Comment:
Wonder if this would be clearer 🤔 

```suggestion
		// Skip circular values that had appeared before
```

Reply:
Hmm "circular" implies that already so I think it should be fine 👍